Analysis/config/2017/sources.cfg

```diff
@@ -85,23 +85,6 @@ sample_type: DY
 #draw_ex: 2b_2Jet kMagenta-5
 #draw_ex: 2b_250ht kPink+7
 #title: DY+{b} b-partons {Jet}Jet
-points: b 0 0 0 0 1 1 1 1 2 2 2 2
```

Comment:
Also remove other not used or commented parts in DY_MC section to avoid confusions in the future